res/values-b+es+r419/strings.xml

```diff
@@ -1111,8 +1111,8 @@ Esta es una lista de cosas * PROHIBIDAS * en las skins personalizadas; si compra
     <string name="ENEMY">ENEMIGO</string>
     <string name="ALLY">ALIADO</string>
     <string name="Rename">Renombrar</string>
-    <string name="Set_friendly_name_">Establecer nombre amistoso </string>
-    <string name="Leave_blank_for_none_">Dejar en blanco para ninguno </string>
+    <string name="Set_friendly_name_">Establecer apodo </string>
+    <string name="Leave_blank_for_none_">Dejar en blanco para eliminar apodo </string>
```

Comment:
Lines 1114 and 1115:

These weren't wrong translations, please leave them as they were